src/vs/vscode.proposed.d.ts

```diff
@@ -2176,4 +2176,16 @@ declare module 'vscode' {
 		}): Disposable;
 	}
 	//#endregion
+
+	//#region https://github.com/microsoft/vscode/issues/105667
+
+	export interface TreeView<T> {
+		/**
+		 * An optional human-readable description that will be rendered in the title of the view.
+		 * Setting the title description to null, undefined, or empty string will remove the title description from the view.
+		 */
+		titleDescription?: string | undefined;
```

Comment:
I think if the value is optional we don't bother with the `| undefined` in API.

Reply:
If I do that, then TS won't let you set `titleDescription` to `undefined` explicitly, which I kind of wanted to keep (vs just setting to an empty string) to remove the description completely.

Reply:
I see, that makes sense.